style.css

```diff
@@ -361,4 +360,82 @@
     width: 45px;
     height: 45px;
   }
+
+  .form-section {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    border-top-left-radius: 70px;
+    background: #6070ff;
+    color: white;
+    padding-top: 40px;
+    margin-bottom: 0;
+    min-height: 80vh;
+    background-image: url("./resources/work-section/images/contact\ form\ shapes-mobile@2x.png");
+    background-size: 55%;
+    background-position: right 30px;
+    background-repeat: no-repeat;
+  }
```

Comment:
Hi @SaveryIV 👋, I requested this change because, on most mobile screen sizes, the background design disappears. It is great to see your work has been approved. 👍